src/Components/Media_query_dimension.js

```diff
@@ -0,0 +1,6 @@
+export const device = {
```

Comment:
Lo sposterei in un altra cartella, magari in una nuova cartella chiamata `assets`